changelogs/unreleased/gh-10363-datetime-handle-timezone.md

```diff
@@ -0,0 +1,5 @@
+## bugfix/datetime
+
+* `datetime` methods now handle timezones properly. Now timezone changes
```

Comment:
>  `datetime` methods now handle timezones properly.

too generic

Isn't it better?  "timestamp is not altered when it was set together with timezone"